.github/workflows/ci_workflows.yml

```diff
@@ -0,0 +1,61 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+  schedule:
+    # Run every Sunday at 04:53 UTC
+    - cron: 53 4 * * 0
+
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Python 3.6 and pytest 4.6 (Linux)
+            os: ubuntu-latest
```

Comment:
`name` is not used. On doctestplus / remotedata I did not put a name because the os/python_version/toxenv combination seems clear enough. If there is a preference for a readable name, that's fine by me, but it should then probably be used to name the build (I guess `    name: ${{ matrix.name }}` near `runs-on`) ?

Reply:
Name is used for display on the left navbar. But if you feel strongly about them, I can remove. I copy pasted from the Travis setup.

Reply:
Lemme know. Thanks!

Reply:
Both are fine, it's just that it would be better to be consistent between repos.
For me the `name` does not add much information.

Reply:
For the sake of moving this forward, I removed the names. See https://github.com/pllim/pytest-openfiles/actions/runs/360370589